dotnetapp-4.7.1/dotnetapp-4.7.1.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I am wondering if we should switch all of our projects to SDK style. That would seem to be a good idea.

Reply:
This is a good idea. Issue : https://github.com/Microsoft/dotnet-framework-docker-samples/issues/18 tracks this issue now. 

@MichaelSimons ,Does this also require updating the build image ?